src/Fantomas/Program.fs

```diff
@@ -462,9 +462,15 @@ Join our Discord community:      https://discord.gg/Cpq9vf8BJH
 
     let asyncRunner = Async.Parallel >> Async.RunSynchronously
 
-    if Option.isSome version then
+    let versionLog =
         let version = CodeFormatter.GetVersion()
-        stdlog $"Fantomas v%s{version}"
+        $"Fantomas v%s{version}"
+
+    if Option.isNone version then
```

Comment:
Nitpick but maybe go for a pattern match here.

Reply:
Mmh, na, I think it reads nicer that way. Also with line 472.